splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceDataDictionary.java

```diff
@@ -1363,13 +1367,81 @@ public void rewriteDescriptors(int catalogNum, long cloned_conglomerate) throws
         }
     }
 
+    public void rewriteColumnDescriptors(int catalogNum, long cloned_conglomerate) throws StandardException {
```

Comment:
we might want to think about moving these upgrade helper functions e.g. create class SpliceDataDictionaryUpgrade . Having those functions that only do upgrade in between other things of SpliceDataDictionary makes the class hard to understand